README.md

```diff
@@ -68,6 +72,67 @@ When you pass down props to the web component, instead of setting attributes lik
 <MyComponent items={elem.items} callback={elem.callback} />
 ```
 
+### Reactifying web components loaded using HTML import
+Custom elements that depend on HTML imports (which are described in another part of the Web Components specification) were previously a bit trickier to integrate into a React project. But this `react-integration` library combined with [the `web-components` Webpack loader](https://github.com/rnicholus/web-components-loader) make the process farily painless. After integrating the Webpack loader into your project, simply `import` the root HTML file of the web component in your React component and use the generated URL to import the Web component using an HTML import in your `render` method. For example:
+
+```jsx
+import React, { Component } from 'react'
+
+import reactify from 'skatejs-react-integration'
+
+const importWcUrl = require('my-web-component/component.html')
+
+class MyWebComponentWrapper extends Component {
+  render() {
+    const MyComponent = reactify('my-web-component')
+
+    return (
+      <span>
+          <link rel='import' href={ importWcUrl } />
+          <MyComponent />
+      </span>
+    )
+  }
+}
+
+export default MyWebComponentWrapper
+```
+
+### Web component attributes
+If the underlying web component you intend to Reactify requires some properties to be set directly on the element as attributes, include an `attr-` prefix on the property name. For example:
+
+```jsx
+<MyComponent attr-data-fo='bar' />
```

Comment:
I've also been playing around with a similar pattern in a separate lib but added support for more special props https://github.com/skatejs/dom-diff#attributes.